src/Microsoft.TestPlatform.PlatformAbstractions/uap10.0/Tracing/PlatformEqtTrace.cs

```diff
@@ -64,7 +64,17 @@ public void WriteLine(PlatformTraceLevel level, string message)
         /// <inheritdoc/>
         public bool InitializeVerboseTrace(string customLogFile)
         {
-            LogFile = Path.GetTempPath() + Path.GetFileNameWithoutExtension(customLogFile).Replace(" ", "_") + ".TpTrace.log";
+            string logFileName = string.Empty;
+            try
+            {
+                logFileName = Path.GetFileNameWithoutExtension(customLogFile.TrimStart('"').TrimEnd('"')).Replace(" ", "_");
+            }
+            catch
+            {
+                logFileName = Guid.NewGuid().ToString();
```

Comment:
Should we not throw error here as this is possible only if customLogFile path has invalid chars. Should we write logs in new file name?

Reply:
We want to disable logging because file name was incorrect, which we constructed?

Reply:
Does this log file path not coming as diag arg value?

Reply:
yes, but we also add bunch of data to it to make it unique.

Reply:
ok. got it. 
So in case user passed some invalid chars in logFilePath, and in result we are creating a log file with different name, is that ok with us?

Reply:
What if user passed valid file name, & we corrupted it?

Reply:
Yes, got that case. Just pointing that there is a scenario in which user given file path will not be respected and user will not be aware of the same (as we will change the log file name and not inform the user)

Reply:
We cannot inform user of that change, this is the first step that we are doing while launching App.
Also your logging should not be based on file name, but unique extension.